themes/10up-theme/functions.php

```diff
@@ -14,12 +14,32 @@
 define( 'TENUP_THEME_INC', TENUP_THEME_PATH . 'includes/' );
 define( 'TENUP_THEME_BLOCK_DIR', TENUP_THEME_INC . 'blocks/' );
 
+$check_hmr =               wp_get_environment_type() === 'local' || 'development';
```

Comment:
```suggestion
$check_hmr =               in_array( wp_get_environment_type(), [ 'local', 'development' ], true );
```

unless I'm missing anything, the current code is going to default to 'development' which is going to be truthy on production envs.

Reply:
Good catch!